PyFry.py

```diff
@@ -12,54 +13,58 @@
       -> Detecting eye coordinates and applying the deepfry eye flare in the center::DONE
 
 '''
-def userInput():
-    #Allowing user to choose the image that has to be deepfried
-    root = tk.Tk()
-    root.withdraw()
-    global filepath
-    filepath = list(root.tk.splitlist(filedialog.askopenfilenames(title="PyFry - Choose Image")))
-    print("picture location = ",filepath)
-
-    
+# def userInput():
+#     #Allowing user to choose the image that has to be deepfried
+#     root = tk.Tk()
+#     root.withdraw()
+#     global filepath
+#     filepath = list(root.tk.splitlist(filedialog.askopenfilenames(title="PyFry - Choose Image")))
+#     print("picture location = ",filepath)
+
+
 def irisCoords(eye):
-    #Finding the center point of the eye using the average outer extremes average of the eyes
-    mid = (eye[0] +eye[3])/2
+    # Finding the center point of the eye using the average outer extremes average of the eyes
+    mid = (eye[0] + eye[3])/2
     mid = (int(mid[0]), int(mid[1]))
     return mid
 
+
 def generateHue(img):
-    #Generating and increasing prominency of red band of the image
+    # Generating and increasing prominency of red band of the image
     img = img.convert('RGB')
-    red = img.split()[0] #(R,G,B)
+    red = img.split()[0]  # (R,G,B)
     red = ImageEnhance.Contrast(red).enhance(2.0)
     red = ImageEnhance.Brightness(red).enhance(1.5)
     red = ImageOps.colorize(red, Colors.RED, Colors.YELLOW)
     img = Image.blend(img, red, 0.77)
-    #Keeping a 100% sharpness value for now, But would probably be better with a higher sharpness value
+    # Keeping a 100% sharpness value for now, But would probably be better with a higher sharpness value
     img = ImageEnhance.Sharpness(img).enhance(150)
     return img
 
+
 def crushAndBack(img):
     img = img.convert('RGB')
-    w,h = img.width, img.height
+    w, h = img.width, img.height
     img = img.resize((int(w ** .95), int(h ** .95)), resample=Image.LANCZOS)
-    img = img.resize((int(w ** .90), int(h ** .90)), resample = Image.BILINEAR)
-    img = img.resize((int(w ** .90), int(h ** .90)), resample = Image.BICUBIC)
-    img = img.resize((w,h), resample = Image.BICUBIC)
+    img = img.resize((int(w ** .90), int(h ** .90)), resample=Image.BILINEAR)
+    img = img.resize((int(w ** .90), int(h ** .90)), resample=Image.BICUBIC)
+    img = img.resize((w, h), resample=Image.BICUBIC)
     return img
+
+
 def addFlare(img):
     ''' Initialising dlib for frontal facial features '''
     flare = Image.open('flare.png')
     detect = dlib.get_frontal_face_detector()
-    predict = dlib.shape_predictor("assets\shape_predictor_68_face_landmarks.dat")
+    face_landmarks_file = "assets/shape_predictor_68_face_landmarks.dat"
```

Comment:
Maybe you can add a dynamic addition of the face_landmarks_file from the GUI, That'll be clean 👍